### ISSUE TITLE
CFG_WITHOUT: parameter not set error

### ISSUE THREAD

**Issue by marcopascha** [rust-lang/rust#25445]
I'm using Linux Mint on VirtualBox. After ```curl -sSf https://static.rust-lang.org/rustup.sh | sh``` process that official site recommend i'm having this error.

Here is the full result of process:
```
sudo curl -s https://static.rust-lang.org/rustup.sh | sudo sh

Welcome to Rust.

WARNING: This script appears to be running as root. While it will work
correctly, it is no longer necessary for rustup.sh to run as root.

This script will download the Rust compiler and its package manager, Cargo, and
install them to /usr/local. You may install elsewhere by running this script
with the --prefix=<path> option.

The installer will run under 'sudo' and may ask you for your password. If you do
not want the script to run 'sudo' then pass it the --disable-sudo flag.

You may uninstall later by running /usr/local/lib/rustlib/uninstall.sh,
or by running this script again with the --uninstall flag.

Continue? (y/N) y

rustup: gpg available. signatures will be verified
rustup: downloading manifest for 'stable'
rustup: downloading toolchain for 'stable'
######################################################################## 100,0%
gpg: Prş 14 May 2015 13:26:24 EEST imzası, RSA anahtarı ve 7B3B09DC kullanıcı kimliği ile yapılmış
gpg: "Rust Language (Tag and Release Signing Key) <rust-key@rust-lang.org>" deki imza iyi
gpg: UYARI: Bu anahtar güven dereceli bir imza ile sertifikalanmamış!
gpg:          Bu imzanın sahibine ait olduğuna dair bir belirti yok.
Birincil anahtar parmak izi: 108F 6620 5EAE B0AA A8DD  5E1C 85AB 96E6 FA1B E5FE
     Yardımcı anahtar parmak izi: C134 66B7 E169 A085 1886  3216 5CB4 A934 7B3B 09DC
rustup: extracting installer
rustup: installing toolchain for 'stable'
/home/khons/.rustup/tmp/tmp-2780-10/rust-1.0.0-x86_64-unknown-linux-gnu/install.sh: 904: /home/khons/.rustup/tmp/tmp-2780-10/rust-1.0.0-x86_64-unknown-linux-gnu/install.sh: CFG_WITHOUT: parameter not set
rustup: command failed: sudo sh /home/khons/.rustup/tmp/tmp-2780-10/rust-1.0.0-x86_64-unknown-linux-gnu/install.sh --prefix=/usr/local
rustup: failed to install toolchain
```

Any solution to offer? Thank you.

**Comment by brson** [comment]
To me this looks like somehow the script was executed with `-u` which makes referencing undefined variables illegal, and the install.sh script does reference undefined variables.

I do not know how this would happen in your scenario. If that's what is happening, there must be some way to set `-u` that I don't know of.

**Comment by brson** [comment]
Moving this to rust-installer.
